app/views/propertyLinking/declaration.scala.html

```diff
@@ -99,69 +100,66 @@ <h1 id="page-header" class="govuk-heading-l">@messages("declaration.title")</h1>
             rows = Seq(
                 SummaryListRow(
                   key = Key(
-                    content = Text(messages("label.address"))
+                    content = valueWithId(messages("label.address"), "address-heading")
                   ),
                     value = Value(
-                        Text(Formatters.capitalisedAddress(model.address))
+                        valueWithId(Formatters.capitalisedAddress(model.address), "address-value")
                     )
                 ),
                 SummaryListRow(
                     key = Key(
-                        content = Text(messages("declaration.checkAnswers.connectionHeading"))
+                        content = valueWithId(messages("declaration.checkAnswers.connectionHeading"), "relationship-heading")
                     ),
                     value = Value(
-                        content = Text(propertyRelationshipCapacityName.fold("")(name => messages(s"declaration.checkAnswers.$name")))
+                        content = valueWithId(propertyRelationshipCapacityName.fold("")(name => messages(s"declaration.checkAnswers.$name")), "relationship-value")
                     ),
                     actions = Some(Actions(
                         items = Seq(
                             ActionItem(
                                 href = s"${controllers.propertyLinking.routes.ClaimPropertyRelationshipController.back().url}",
                                 content = Text(messages("declaration.checkAnswers.change")),
                                 visuallyHiddenText = Some(messages("declaration.checkAnswers.connectionHeading")),
-                                attributes = Map("id" -> "propertyRelationshipChange")
+                                attributes = Map("id" -> "relationship-change")
                             )
                         )
                     ))
                 ),
                 if(!isEarliestStartDateInFuture){
                 SummaryListRow(
                     key = Key(
-                        content = Text(messages("declaration.checkAnswers.startDateHeading")),
-                        classes = "start-date-heading"
+                        content = valueWithId(messages("declaration.checkAnswers.startDateHeading"), "start-date-heading"),
                     ),
                     value = Value(
-                        content = Text(
-                            propertyOwnershipFromDate.fold(messages("declaration.checkAnswers.startDate.before2017"))(_.format(DateTimeFormatter.ofPattern("d MMMM yyyy")))
-                        )
+                        content = valueWithId(propertyOwnershipFromDate.fold(messages("declaration.checkAnswers.startDate.before2017"))(_.format(DateTimeFormatter.ofPattern("d MMMM yyyy"))), "start-date-heading")
```

Comment:
I think this should be `start-date-value`, just running the test against this branch and it can't find the value :)

Reply:
thanks, fixing now